odoo/fields.py

```diff
@@ -2350,10 +2350,17 @@ def write(self, records, value):
         records.env.cache.update(records, self, [cache_value] * len(records))
 
     def _image_process(self, value):
-        return image_process(value,
+        if self.readonly and not self.max_width and not self.max_height:
```

Comment:
`... and not self.verify_resolution:`

Reply:
Not sure about that; the above condition is enough for me.